install

```diff
@@ -0,0 +1,28 @@
+-- Install the program by rewriting dofiles, and creating a link to the program
+
+local dest_dir = shell.resolve(select(..., 1) or '_build')
+
+print("Installing to %s...":format(dest_dir))
+
+local copyAndFix(src_file, src_file)
+	local r = fs.open(src_file, 'r')
+	local data = r.readAll()
+	r.close()
+	local w = fs.open(dest, 'w')
+	data = data:gsub('dofile%("', 'dofile("'..dest_dir..'/')
+	w.write(data)
+	w.close()
+end
+
+for _, folder in ipairs({'api', 'programs'})
```

Comment:
I think this would be better served as a table up by `FILES` instead of hardcoded here.